PVTPackage/source/MultiphaseSystem/NegativeTwoPhaseMultiphaseSystem.hpp

```diff
@@ -54,6 +64,8 @@ class NegativeTwoPhaseMultiphaseSystem : public CompositionalMultiphaseSystem
   NegativeTwoPhaseFlash m_negativeTwoPhaseFlash;
 
   NegativeTwoPhaseFlashMultiphaseSystemProperties m_ntpfmsp;
+
+  std::vector< NegativeTwoPhaseFlashMultiphaseSystemProperties > m_batchProps;
```

Comment:
And this should not exist without openmp neither.

Reply:
I removed this